nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1704,17 +1704,32 @@ For more information, see the <<toolkit-guide.adoc#nifi_CLI,NiFi CLI>> section i
 === Flow Election
 When a cluster first starts up, NiFi must determine which of the nodes have the
 "correct" version of the flow. This is done by voting on the flows that each of the nodes has. When a node
-attempts to connect to a cluster, it provides a copy of its local flow to the Cluster Coordinator. If no flow
+attempts to connect to a cluster, it provides a copy of its local flow and its users, groups, and policies (user permissions), to the Cluster Coordinator. If no flow
```

Comment:
We should probably clarify that this capability is only for implementations that support configuration in NiFi. These are identified by implementing `ConfigurableAccessPolicyProvider`. Other implementations that are not configurable like the `LdapUserGroupProvider` or the `ManagedRangerAuthorizer`(which provides an implementation of `AccessPolicyProvider`) would not follow these same rules for users, groups, and policies.